protocol/gabriel.proto

```diff
@@ -6,20 +6,42 @@ option java_package = "edu.cmu.cs.gabriel.network";
 option java_outer_classname = "Protos";
 
 message Input {
-  string update_id = 1;
-  enum Type {
-    IMAGE = 0;
-    VIDEO = 1;
-    AUDIO = 2;
-    ACCEL = 3;
-  }
-  Type type = 2;
-  bytes data = 3;
-  string style = 4;
-  string filename = 5;
+    int32 frame_id = 1;
```

Comment:
I think we should keep this a string. We don't want to run into overflow issues and we aren't doing math with the frame_id. 

If this has to be a numeric type, we should at least use uint64.

Reply:
Numeric type makes more sense to me; doing str -> int parsing will probably expose us to bugs in the future. I agree with it being an uint64 though.

Reply:
I suspect the Overflow issue will bite people more than str->int parsing will.

Reply:
uint64 should be more than enough for that. Even at a constant rate of 1 frame per millisecond, it'll take 584942417 years before we run into overflow issues.

Frames in a year at a rate of 1 frame/ms: 1000 * 3600 * 24 * 365 = 31536000000
Max(uint64) = 18,446,744,073,709,551,615

18,446,744,073,709,551,615 / 31536000000 =  584942417.355

Reply:
My thought is that, regardless of the data type chosen for the protobuf definition, it will be enforced by implementation for whatever bindings we build it for. If it is Python and you try to do .frame_id = 1234 then a TypeError will be thrown. The protobuf definition is easy to consult if you want to know what datatype something is so I believe the risk of errors due to inferring the wrong data type would be very minimal. As such, I would recommend making it whatever type is the most useful. Since we are likely using this value for comparison it may be more useful as an integer (unsigned or otherwise) than a string. Just my two cents as they say.